backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -73,9 +93,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     } else {
       // Match not found, insert participant entity into datastore
       Entity participantEntity = new Entity("Participant");
-      participantEntity.setProperty("ldap", ldap);
-      participantEntity.setProperty("timeAvailableUntil", timeAvailableUntil);
-      participantEntity.setProperty("timezone", timezone);
+      participantEntity.setProperty("username", username);
+      participantEntity.setProperty("startTimeAvailable", startTimeAvailable);
+      participantEntity.setProperty("endTimeAvailable", endTimeAvailable);
       participantEntity.setProperty("duration", duration);
       participantEntity.setProperty("timestamp", timestamp);
```

Comment:
since each of these keys are referenced twice, there's a possibility to introduce bugs when one instance is changed and the other isn't. For example if "ldap" was updated here but not below. 

To guard against this, I'd recommend extracting string constants for each of these keys. Then when you want to change the key, it only has to be updated in a single place.

Reply:
What if I used newParticipant.getUsername(), .getStartTimeAvailable(), etc?

Reply:
Done.